src/DataDepsGenerators.jl

```diff
@@ -158,6 +167,10 @@ function format_authors(authors::Vector)
     end
 end
 
+function handle_null(attr::Any)
+    attr != nothing? attr : missing
+end
```

Comment:
This belongs in `utils.jl` next to `lift`
And it should be written using dispatch on the type of its input.
You know what I mean?

Reply:
Maybe it should be called `miss_null`